youtube_dl/extractor/common.py

```diff
@@ -2194,7 +2194,11 @@ def geo_verification_headers(self):
         return headers
 
     def _generic_id(self, url):
-        return compat_urllib_parse_unquote(os.path.splitext(url.rstrip('/').split('/')[-1])[0])
+        id = compat_urllib_parse_unquote(os.path.splitext(url.rstrip('/').split('/')[-1])[0])
+        # If our id matches a stopword, use the 2nd-to-last URL path component instead
+        if id in ['story']:
+            id = compat_urllib_parse_unquote(os.path.splitext(url.rstrip('/').split('/')[-2])[0])
```

Comment:
`story` may be a 5-character generated id even though the probability of this outcome is close to zero. Even if we stick with this stopword approach why only `story`? There lots of other words and lots of other languages. It's just not feasible.
In case of generic extractor the id is more like a dummy placeholder. It's not used to identify something and you can't make it meaningful for all the cases while in generic extractor anyway.